near/src/runtime.rs

```diff
@@ -1050,9 +1066,9 @@ mod test {
         assert_eq!(account.staked, 2 * TESTING_INIT_STAKE);
         assert_eq!(account.amount, TESTING_INIT_BALANCE - TESTING_INIT_STAKE * 5);
 
-        // Send invalid transaction to see if the rollback of the state affects the validator rewards.
-        let invalid_transaction = stake(0, &new_signer, &new_validator, TESTING_INIT_STAKE * 2);
-        env.step_default(vec![invalid_transaction]);
+        // NOTE: The Runtime doesn't take invalid transactions anymore (e.g. one with a bad nonce),
+        // because they should be filtered before producing the chunk.
+        env.step_default(vec![]);
```

Comment:
please keep invalid tx, so we test rollback inside the state transition after epoch change.